application/src/main/java/org/thingsboard/server/controller/Lwm2mController.java

```diff
@@ -46,7 +46,7 @@
     @ResponseBody
     public ServerSecurityConfig getLwm2mBootstrapSecurityInfo(@PathVariable("isBootstrapServer") boolean bootstrapServer) throws ThingsboardException {
         try {
-            return lwM2MServerSecurityInfoRepository.getServerSecurityInfo(bootstrapServer);
+            return deviceCredentialsService.getServerSecurityInfo(bootstrapServer);
```

Comment:
rename to getLwm2mBootstrapSecurityInfo. How can someone guess that getServerSecurityInfo is related to LwM2M Bootstrap Server?